### PR TITLE
Backup hourly ClusterResources and all Namespaces

### PR DESCRIPTION
**Pull request by scheeles**
**Documentation**: tbd

**Does this PR introduce a user-facing change?**:
```release-note
Create an hourly schedule Velero backup for all namespaces and cluster resources
```

### CODE REVIEW

File: config/backup/velero/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: velero
-version: 1.2.1
+version: 1.3.2
```
Comment (kron4eg) [inline config/backup/velero/Chart.yaml:3]: did you wanted to update velero to the latest release?

Comment (scheeles) [reply config/backup/velero/Chart.yaml:3]: now it is correct

**Review by kron4eg — APPROVED** [review]

**Pull request merged**